### ISSUE TITLE
False positive on comment after delete operator

### ISSUE THREAD

**Issue by mbrodala** [martin-helmich/typo3-typoscript-lint#33]
Given the following TypoScript:

```
foo > # Something
```

The linter shows the following error:

> Parse error: Unexpected token RVALUE after delete operator (expected line break).

However, comments after the delete operator (although unusual) are perfectly fine.

**Comment by martin-helmich** [comment]
This is actually not an issue in the linter, but in the parser (which is a [separate library](https://github.com/martin-helmich/typo3-typoscript-parser)). I migrated this report there.

**Comment by mbrodala** [comment]
You're right, thanks.
